Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Paged/ContentBlockerScriptHandler.swift

```diff
@@ -66,7 +66,7 @@ extension ContentBlockerHelper: TabContentScript {
       Task { @MainActor in
         let isPrivateBrowsing = PrivateBrowsingManager.shared.isPrivateBrowsing
         let domain = Domain.getOrCreate(forUrl: currentTabURL, persistent: !isPrivateBrowsing)
-        if let shieldsAllOff = domain.shield_allOff, Bool(truncating: shieldsAllOff) {
+        guard !domain.areAllShieldsOff else {
```

Comment:
Ok I can change this. 

Though I kind of like guards because you know it will return if the condition is not met and you know that it doesn't continue execution. Whereas with an `if` you have to look at the content of the `if`. 
Though I agree that in a simple `if` is pretty easy to see the return and you don't need to negate the condition like I do here.